project/wei0107/README.rst

```diff
@@ -91,14 +149,23 @@ Schedule
 
 Development timeline (8 weeks):
 
-- **Week 1 (10/06)**: Repository setup, project skeleton (C++ core + Python binding). Implement basic image I/O (load/save).
-- **Week 2 (10/13)**: Implement convolution framework. Add Average and Gaussian filters with unit tests.
-- **Week 3 (10/20)**: Implement Median and Bilateral filters. Test with noisy images.
-- **Week 4 (10/27)**: Implement color and tone adjustments (Grayscale, Sepia, Invert, Brightness/Contrast).
-- **Week 5 (11/03)**: Implement sharpening and effects (Sharpen, Emboss). Begin work on Cartoon effect (smoothing + edge detection).
-- **Week 6 (11/10)**: Implement geometric operations (Resize, Rotate, Flip, Crop). Add OpenMP parallelization for convolution.
-- **Week 7 (11/17)**: Expand testing framework (pytest + Catch2). Add demo examples with everyday photos. Draft documentation.
-- **Week 8 (11/24)**: Final polishing: functional tests, performance benchmarks (single-thread vs OpenMP), and prepare presentation/demo.
+- **Week 1 (10/06)**: Repository setup, project skeleton (C++ core + Python
+  binding). Implement basic image I/O (load/save).
+- **Week 2 (10/13)**: Implement convolution framework. Add Average and
+  Gaussian filters with unit tests.
+- **Week 3 (10/20)**: Implement Median and Bilateral filters. Test with
+  noisy images.
+- **Week 4 (10/27)**: Implement color and tone adjustments (Grayscale,
+  Sepia, Invert, Brightness/Contrast).
+- **Week 5 (11/03)**: Implement sharpening and effects (Sharpen, Emboss).
+  Begin work on Cartoon effect (smoothing + edge detection).
+- **Week 6 (11/10)**: Implement geometric operations (Resize, Rotate, Flip,
+  Crop). Add OpenMP parallelization for convolution.
+- **Week 7 (11/17)**: Expand testing framework (pytest + Catch2). Add demo
+  examples with everyday photos. Draft documentation. Set up CI pipeline
+  (GitHub Actions) for automated build and test execution.
+- **Week 8 (11/24)**: Final polishing: functional tests, performance
+  benchmarks (single-thread vs OpenMP), and prepare presentation/demo.
```

Comment:
The time usually is insufficient for multi-threading, but making it as a stretch goal is fine.